<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="39695" started-at="2015-07-28T12:45:30Z" finished-at="2015-07-28T12:46:10Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="39695" started-at="2015-07-28T12:45:30Z" finished-at="2015-07-28T12:46:10Z">
      <class name="com.ccautomation.testPackage.SeriesLinks">
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:com.ccautomation.testPackage.SeriesLinks@80f252]" name="beforeClass" is-config="true" duration-ms="2477" started-at="2015-07-28T18:15:30Z" finished-at="2015-07-28T18:15:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="FAIL" signature="Addserieslinks()[pri:0, instance:com.ccautomation.testPackage.SeriesLinks@80f252]" name="Addserieslinks" duration-ms="35999" started-at="2015-07-28T18:15:33Z" finished-at="2015-07-28T18:16:09Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.ccautomation.testPackage.SeriesLinks.Addserieslinks(SeriesLinks.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Addserieslinks -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:com.ccautomation.testPackage.SeriesLinks@80f252]" name="afterClass" is-config="true" duration-ms="1207" started-at="2015-07-28T18:16:09Z" finished-at="2015-07-28T18:16:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- com.ccautomation.testPackage.SeriesLinks -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
